metadata.yaml

```diff
@@ -18,7 +23,7 @@ missions:
   name: Health Check
   description: The Health Check mission demonstrates the health check pattern. When you deploy an application, its important to know if it is available and if it can start handling incoming requests. Implementing the health check pattern allows you to monitor the health of an application, which includes if an application is available and whether it is able to service requests.
   metadata:
-    level: foundational
+    level: foundational    
```

Comment:
Ugh...

Reply:
@cescoffier mind not changing this line? ;)